main.go

```diff
@@ -630,6 +620,30 @@ func main() {
 		l.Panicf("Server problem: %+v", err)
 	}
 
+	// handle unix signals
+	terminationSignals := make(chan os.Signal, 1)
+	signal.Notify(terminationSignals, unix.SIGTERM, unix.SIGINT)
+	updateSignals := make(chan os.Signal, 1)
+	signal.Notify(updateSignals, unix.SIGHUP)
+	go func() {
+		for {
```

Comment:
this `for` is not required

Reply:
It is required – SIGHUP can be sent several times

Reply:
oh, ok